main/tests/server/src/com/google/refine/importers/FixedWidthImporterTests.java

```diff
@@ -159,40 +165,41 @@ public void readColumnHeader() throws Exception {
     }
 
     @Test
-    public void testDeleteEmptyColumns() throws Exception {
-        StringReader reader = new StringReader(SAMPLE_ROW + "\nTooShort");
+    public void testDeleteEmptyColumns() throws IOException {
+        String filename = "fixed-width-test-file-header-and-sample-row-with-empty-column.txt";
+        List<String> lines = List.of(SAMPLE_ROW, "012345green...."); // add blank column
+        List<ObjectNode> fileRecords = prepareFileRecords(filename, lines);
 
-        ArrayNode columnWidths = ParsingUtilities.mapper.createArrayNode();
-        // Set up blank column in project
-        JSONUtilities.append(columnWidths, 6);
-        JSONUtilities.append(columnWidths, 0);
-        JSONUtilities.append(columnWidths, 5);
-        JSONUtilities.append(columnWidths, 0);
-        JSONUtilities.append(columnWidths, 3);
-        whenGetArrayOption("columnWidths", options, columnWidths);
+        ObjectNode options = ParsingUtilities.mapper.createObjectNode();
+        JSONUtilities.safePut(options, "limit", -1);
+        JSONUtilities.safePut(options, "skipDataLines", 0);
+        JSONUtilities.safePut(options, "ignoreLines", 0);
+        JSONUtilities.safePut(options, "headerLines", 1);
 
-        ArrayNode columnNames = ParsingUtilities.mapper.createArrayNode();
-        columnNames.add("Col 1");
-        columnNames.add("Col 2");
-        columnNames.add("Col 3");
-        columnNames.add("Col 4");
-        columnNames.add("Col 5");
-        columnNames.add("Col 6");
```

Comment:
It looks like this test case supplied some custom column names. Wouldn't it be worth retaining them? Or is this functionality adequately tested elsewhere?